tools/appliance/builtin/template.json

```diff
@@ -23,8 +23,8 @@
       "disk_interface": "virtio",
       "net_device": "virtio-net",
 
-      "iso_url": "http://mirror.nbrc.ac.in/centos/7/isos/x86_64/CentOS-7-x86_64-Minimal-1708.iso",
-      "iso_checksum": "5848f2fd31c7acf3811ad88eaca6f4aa",
+      "iso_url": "http://mirror.nbrc.ac.in/centos/7/isos/x86_64/CentOS-7-x86_64-Minimal-1808.iso",
```

Comment:
I did not check this, looks like I made a mistake in merging this @khos2ow can you send a new PR to fix the iso_url, it ends with `1804` and not 1808.

Reply:
oh yes! copy paste error!

Reply:
the checksum is _correct_ and based on the official repo sha1 hash, but either the type needs to be changed from `md5` to `sha1` or manually calculate the checksum based on `md5`. We can follow the discussion in #2775